### PR TITLE
Feature - auto populate

### PR DESCRIPTION
**Pull request by ulises-aguilar**
* Buttons "From" and "To" in the relationship section can now auto populate class names.

### CODE REVIEW

File: src/main/java/org/project/Controller/MainController.java [hunk 1]
```diff
@@ -124,7 +137,18 @@ public void handleSetClassName(ActionEvent event) {
...
+
+            int fromIndex = fromComboBox.getItems().indexOf(currentName);
+            int toIndex = toComboBox.getItems().indexOf(currentName);
+
+            if (fromIndex >= 0) {
+                fromComboBox.getItems().set(fromIndex, newName);
+            }
+            if (toIndex >= 0) {
+                toComboBox.getItems().set(toIndex, newName);
+            }
```
Comment (rudmad13) [inline src/main/java/org/project/Controller/MainController.java:151]: This does add the classes as they are being created. But one bug that I found is that it won't delete the option from the combo box as a class in the diagram is deleted

**Review by rudmad13 — CHANGES_REQUESTED** [review]
Please look at line 151 for the bug that I found. Please fix and submit another request. Please and thank you

File: src/main/java/org/project/Controller/MainController.java (hunk 1 above)
Comment (ulises-aguilar) [reply src/main/java/org/project/Controller/MainController.java:156]: I have fixed the bug. Now when the user deletes a class it deletes the class name in the combo box "From" and "To" as well.

**Review by rudmad13 — APPROVED** [review]
No bugs found on my end. Will merge into developer

**Pull request merged**